sources/Foundation/Types/Types.h

```diff
@@ -56,6 +56,7 @@ struct FourCC {
     SampleInstrumentEnd = 6,
     SampleInstrumentTable = 117,
     SampleInstrumentTableAutomation = 60,
+    SampleInstrumentSlices = 202,
```

Comment:
Please use the next available number and add this usage at line 162. I think this is my fault as I incorrectly allocated a couple of new consts as 200, 201 but I have since fixed that in `master` so the next available number should be `151.